charts/observability-pipelines-worker/values.yaml

```diff
@@ -24,19 +24,28 @@ datadog:
   # datadog.configKeyExistingSecret -- Specify a preexisting Secret that has your configuration key instead of
   # creating a new one. The value must be stored under the `config-key`.
   configKeyExistingSecret: ""
+  # datadog.pipelineID -- Specify your Datadog Observability Pipelines pipeline ID
+  pipelineID:  # <DD_OP_PIPELINE_ID>
+  # datadog.pipelineIDExistingSecret -- Specify a preexisting Secret that has your pipeline ID instead of
+  # creating a new one. The value must be stored under the `pipeline-id`.
+  pipelineIDExistingSecret: ""
```

Comment:
```suggestion
  pipelineIdExistingSecret: ""
```

Nit/pedantic

Reply:
Adjusted in [7a5e07c](https://github.com/DataDog/helm-charts/pull/952/commits/7a5e07c5b852e9393a9f49a75395fe08597c52dd)